src/core/shards/flow.cpp

```diff
@@ -456,20 +456,18 @@ struct Maybe : public BaseSubFlow {
       if (_silent) {
         spdlog::set_level(spdlog::level::off);
       }
-      DEFER({
-        if (_silent) {
+      auto state = _shards.activate(context, input, output);
+      if (_silent) {
 #ifdef NDEBUG
```

Comment:
Might be a good opportunity to simplilfy this, just store `spdlog::get_level()` and restore it after activate